src/annotator/anchoring/pdf.coffee

```diff
@@ -132,7 +135,7 @@ anchorByPosition = (page, anchor, options) ->
 # Returns a `Promise<Range>` for the location of the quote.
 findInPages = ([pageIndex, rest...], quote, position) ->
   unless pageIndex?
-    return Promise.reject('quote not found')
+    return Promise.reject(new Error('Quote not found'))
```

Comment:
The convention is that Promises should always be rejected with an Error or Error-like object. Why? Because Promise handlers can then rely on this. In tests for example, Mocha has standard logic for formatting Errors when an async test fails with a promise rejection.